test/reduce.js

```diff
@@ -1,5 +1,6 @@
 var R = require('..');
 var eq = require('./shared/eq');
+var _xwrap = require('../src/internal/_xwrap');
```

Comment:
Ugly, yes, but I wasn't able to find a public fn that "lifts" a normal reducing function `a -> b -> a` into a "transformer" (an object that contains the step, result and init cases of a). 

Transducers (e.g. `take(2)`) expect to be applied to a transformer.